build.sbt

```diff
@@ -39,3 +39,8 @@ lazy val scalaz = (project in file("scalaz"))
   .settings(moduleName := "github4s-scalaz")
   .settings(scalazDependencies: _*)
   .dependsOn(github4sJVM)
+
+lazy val catsEffect = (project in file("cats-effect"))
+  .settings(moduleName := "github4s-cats-effect")
+  .settings(catsEffectDependencies: _*)
+  .dependsOn(github4sJVM)
```

Comment:
I don't know if `cat-effects` is already a crossed-project with Scalajs (but it will be likely). If that's already the case, we could make `catsEffect` module as `crossProject` too.